person.rb

```diff
@@ -29,4 +30,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- You have not properly implemented the has-many relationship between the `Person` class and the `Rental` class. Remember a `person` can have many `rentals` i.e the `add_rental` method should look like this
``` 
def add_rental( date, book)
   Rental.new(date, book, self)
end
```